core/templates/dev/head/pages/exploration_editor/EditorNavigationDirective.js

```diff
@@ -77,11 +77,18 @@ oppia.directive('editorNavigation', [
                   var explorationId = (
                     ContextService.getExplorationId());
 
-                  $scope.beginTutorial = function() {
+                  $scope.beginEditorTutorial = function() {
                     SiteAnalyticsService
                       .registerOpenTutorialFromHelpCenterEvent(
                         explorationId);
-                    $uibModalInstance.close();
+                    $uibModalInstance.close('editor');
```

Comment:
Yeah this was made the following way so that modal the corresponding button on the help modal could trigger the correct tutorial in the resulting promise. So yeah we can move this to oppia.constant and have the respective button functions load the correct tutorial.

Reply:
So I will make something like editor_tutorial_mode and translation_tutorial_mode as local variables.

Reply:
Modified to a camelcase version of editor_tutorial_mode and translation_tutorial_mode. Done